scripts/golinter.sh

```diff
@@ -31,6 +31,18 @@ do
     fi
 done
 
+for i in "${goModules[@]}"
+do
+    echo ">>>Checking $i with goimports"
+    OUTPUT="$(goimports -l ./$i || true )"
+    if [[ $OUTPUT ]]; then
+        echo "The following files contain goimports errors"
+        echo $OUTPUT
+        echo "The goimports command 'goimports -l -w' must be run for these files"
+        exit 1
+    fi
+done
+
```

Comment:
something something shell magic to merge these two arrays and remove the duplication.

needs research, I don't know off the top of my head.